demos/dermatology/tests/chartFactory.js

```diff
@@ -412,10 +412,43 @@
             simple: function (callback) {
                 legacyRequire(["test/DataFactory", "src/chart/Pie"], function (DataFactory, Pie) {
                     callback(new Pie()
-                        .columns(DataFactory.TwoD.subjects.columns)
-                        .data(DataFactory.TwoD.subjects.data)
+                        .columns(["Subject", "Students"])
+                        .data([
+                            ["Structured Computer Architecture", 390],
+                            ["Intro to Microprocessor Systems", 450],
+                            ["CAD-Based Computer Design", 350],
+                            ["Computer Operating Systems", 200],
+                            ["Design & Analysis of Algorithms", 200],
+                            ["Stochastic Models for Computer Science", 190],
+                            ["Data Structures & Algorithm Analysis", 40],
+                            ["Calculus with Analytic Geometry I", 39],
+                            ["Calculus with Analytic Geometry II", 37],
+                            ["Theory and Philosophy of Computation", 38],
+                            ["Analysis of Algorithms", 32],
+                            ["Theory and Philosophy of Computation", 35],
+                            ["Randomized Algorithms", 41],
+                            ["Queueing Theory", 42],
+                        ])
                     );
                 });
+            },
+            many_small: function (callback) {
+                legacyRequire(["test/DataFactory", "src/chart/Pie"], function (DataFactory, Pie) {
+                    const pie = new Pie()
+                        .startAngle(0.3)
+                        .columns(DataFactory.TwoD.subjects.columns)
+                        .data(Array(26).fill(1).map((n, i) => {
+                            return [Array(20).fill(String.fromCharCode(65 + i)).join(''), i + 1];
+                        }))
+                        ;
+                    setInterval(function () {
+                        pie
+                            .startAngle(pie.startAngle() + 0.3)
+                            .render()
+                            ;
+                    }, 1500)
```

Comment:
You can lower to 300 for nicer affect,

Reply:
Change to 300

Reply:
Nevermind, one of the transition calls was causing the labels to transition twice.